pkg/components/storage.go

```diff
@@ -75,6 +75,11 @@ func startCSIPlugin(cfg *config.MicroshiftConfig, kubeconfigPath string) error {
 		}
 	)
 
+	if err := lvmd.LvmSupported(); err != nil {
+		klog.Warning("skipping CSI deployment: %v", err)
```

Comment:
Nothing to hold this PR if we don't want to touch it, but it's missing `f`: `klog.Warningf` so the log looks like:
```
skipping CSI deployment: %vfailed to find 'vgs' command line tool: exec: "vgs": executable file not found in $PATH
```
I'd say we could leave it as is in 4.13 and make a small fix in `main` (or even just fix it when touching this file in another PR)

Reply:
Yeah, Jon caught that and fixed it in this PR, https://github.com/openshift/microshift/pull/1598 seems the bot didn't pick up his cherry pick.

Reply:
Well it didn't pick it up because the timing is off, this needs to go in first before we can cherry-pick that one

Reply:
Yeah, i'll have that backported rather than fix the the bot-generated PR.